### DISCUSSION TITLE
Change number of columns in a filter

### DISCUSSION THREAD

**Discussion by eaziweb95** [filamentphp/filament#2748] [Help] [answered]
I've added this method in the resource file to have 3 columns in the filter but nothing changes

    protected function getTableFiltersFormColumns(): int {
        return 3;
    }

There is no example in the demo site

**Comment by danharrin** [comment, marked as answer]
The method needs to go on the List page, not the Resource

**Reply by eaziweb95** [reply]
Thanks a lot !
